cookbooks/aws-parallelcluster-slurm/recipes/config/config_head_node.rb

```diff
@@ -15,6 +15,29 @@
 # OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Copy pcluster config generator and templates
+remote_directory "#{node['cluster']['scripts_dir']}/slurm" do
+  source 'head_node_slurm/slurm'
+  mode '0755'
+  action :create
+  recursive true
+end
```

Comment:
Why did we move this block to the top of the recipe?

Reply:
It looks like you need to sort this out, because recently this was moved to a new recipe called `'aws-parallelcluster-slurm::config_head_node_directories'`

Reply:
OK, it was done.